src/geometry/ArborX_DetailsAlgorithms.hpp

```diff
@@ -573,6 +574,165 @@ struct intersects<PointTag, TriangleTag, Point, Triangle>
   }
 };
 
+template <typename Box, typename Triangle>
+struct intersects<BoxTag, TriangleTag, Box, Triangle>
+{
+  KOKKOS_FUNCTION static constexpr bool apply(Box const &box,
+                                              Triangle const &triangle)
+  {
+    // Based on the Separating Axis Theorem
+
+    // Test bounding boxes, i.e., normals of the box
```

Comment:
I don't really understand the comment about normals. How about this:
```suggestion
    // Do a quick test of bounding boxes
```

Reply:
We need to test 13 axes, 3 for the normals of box, 1 for the normal of the triangle, and 9 for the crossproducts between the 3 edge-directions of the box and the 3 edge-directions of the triangle.